readme.md

```diff
@@ -1,42 +1,45 @@
-# Microsoft Azure IoT SDKs and libraries for C
+# Azure IoT C SDKs and Libraries
 
 This repository contains the following:
-* **Microsoft Azure IoT Hub device SDK for C** to connect devices running C code to Azure IoT Hub
-* **Microsoft Azure Iot Hub Device Provisioning Service client SDK** for enrolling devices with Device Provisioning Services and managing enrollments lists.
-* **Microsoft Azure IoT Hub service SDK for C** to interface with an Azure IoT Hub service instance from a back-end C application
-* **Serializer library for C** to help serialize and deserialize data on your device.
+* **Azure IoT Hub Device C SDK** to connect devices running C code to Azure IoT Hub.
+* **Azure IoT Hub Device Provisioning Service Client SDK** for enrolling devices with [Azure IoT Device Provisioning Services](https://docs.microsoft.com/azure/iot-dps/) and managing enrollments lists.
+* **Azure IoT Hub Service C SDK** to interface with an Azure IoT Hub service instance from a server-side C application.
+* **Serializer Library for C** to help serialize and deserialize data on your device.
+
+## Packages and Libraries
+  The simplest way to use the Azure IoT SDKs is to use the following packages and libraries:
+  * Linux (Ubuntu 14.04, 15.04, 15.10, 16.04): [Device SDK on apt-get](./iothub_client/readme.md#aptgetpackage)
```

Comment:
We currently only build for 14.04 and 16.04.  How many / far back to support is an ongoing discussion but for now please remove 15.04 and 15.10.